bin/run.sh

```diff
@@ -1,3 +1,3 @@
 # turning off slack for now!
 # invoke notify
-gunicorn -w 2 openfecwebapp.app:app
+gunicorn --log-level debug -k gevent -w 2 openfecwebapp.app:app
```

Comment:
Just a note for others - we're going to have this set for the time being to watch what's happening and try and glean more info, then shut it off.